src/index.html

```diff
@@ -1,12 +1,43 @@
-<!DOCTYPE html>
+ <!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+
+  <body class="calendar calendar--start-day calendar--month-length">
```

Comment:
```suggestion
  <body class="calendar calendar--start--day calendar--month-length">
```
day???
what day? Monday, Friday or another one?

Reply:
same with length
if we change the modifier here - your calendar should apply changes